harness/tests/test_ipc.py

```diff
@@ -237,20 +237,35 @@ def make_distributed_context(rank: int, cross_rank: int, local_rank: int) -> Any
 
 
 class TestPIDServer:
+    @staticmethod
+    def _worker_proc(
+        addr: Union[str, int, Tuple[str, int]],
```

Comment:
might as well make the `addr` just a single port, since that's all we invoke it with